akka-http-core/src/main/scala/akka/http/javadsl/ConnectionContext.scala

```diff
@@ -79,14 +93,19 @@ abstract class HttpsConnectionContext extends akka.http.javadsl.ConnectionContex
   override final def isSecure = true
 
   /** Java API */
+  @Deprecated @deprecated("here for binary compatibility", since = "10.2.0")
```

Comment:
Do we have replacements for those methods? I guess we say, you have to provide a ready configured SSLContext / SSLEngine factory from now on, right? Might make sense to add that to the deprecation methods (only needed for the constructors).

Reply:
I think https://github.com/akka/akka-http/pull/2983/files#diff-f7c7e05e650a31f0e256ceaa3bca12a3R127 is the only one, made it a bit more complete